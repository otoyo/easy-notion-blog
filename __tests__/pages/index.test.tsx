import { render } from '@testing-library/react'
import RenderPage from '../../src/pages/index'

jest.mock('next/router', () => ({
  useRouter() {
    return {
      asPath: '/',
      pathname: '/',
    }
  },
}))

describe('RenderPage', () => {
  it('renders the page unchanged', () => {
    const { container } = render(<RenderPage />)
    expect(container).toMatchSnapshot()
  })
})
