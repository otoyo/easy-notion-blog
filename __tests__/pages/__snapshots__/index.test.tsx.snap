// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RenderPage renders the page unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <div>
      <h2>
        Welcome!
      </h2>
      <p>
        Your easy-notion-blog deployed successfully!
      </p>
      <p>
        Have fun!
      </p>
      <p>
        easy-notion-blog powered by
         
        <a
          href="https://github.com/otoyo/easy-notion-blog"
          rel="noopener"
          target="_blank"
        >
          otoyo/easy-notion-blog
        </a>
      </p>
    </div>
  </div>
</div>
`;
